src/components/layout/SearchInput.vue

```diff
@@ -4,17 +4,17 @@
     :class="{ sided, focused }"
   >
     <input
-      v-model="query"
+      :value="value"
       type="text"
       :placeholder="placeholder"
       @focus="focused = true"
       @blur="focused = false"
       @input="search($event.target.value)"
     >
     <ButtonPlain
-      v-if="query.length"
+      v-if="value.length"
       :title="$t('views.TipList.Clear')"
-      @click="clear"
+      @click="search('')"
```

Comment:
```suggestion
      @click="$emit('input', '')"
```
To don't have clearing debounced.